numbers/index.js

```diff
@@ -78,20 +81,26 @@ export const roundDown = (num) => {
   converted to a number
 */
 export const convertToInt = (str) => {
-
+  if (Number.isNaN(Number(str)) || typeof str !== 'string') {
+    return false;
+  }
+  return Math.round(Number(str)); // this took me by far the longest
 }
 
 /*
   I expect this function to return true if a number is odd, false if it is even
 */
 export const isOdd = (num) => {
-
+  if (num % 2 === 0) {
+    return false;
+  }
+  return true;
 }
 
 /*
   I expect this function to take the numnber and round it to the decmials
   provided by the decimals argument
 */
 export const variableDecimalPlace = (num, decimals) => {
-
+  return Number(parseFloat(num).toFixed(decimals));
```

Comment:
NICE! Good solution! 

Couple of things to note here: when you call parseFloat() you are referring to what's called a "global function" because you can access it anywhere in a JS file (therefore it's considered global).

Usually using global functions is discouraged as global functions go against the notion of functional programming (where you write functions that have jobs and everything is self contained within 'modules' that do their own specific jobs).

You can access parseFloat from the Number class
```js
// avoid if possible... (this is a global function!)
const a = parseFloat('12.3');

// better!
const a = Number.parseFloat('12.3');
```

Also, a small improvement consideration - what if the result from `parseFloat()` is `NaN`? will that cause the function to error when you try and call `.toFixed()`?

Maybe write a test case for this to also fail gracefully and update

Still, great job on the original solution 👍